website/sidebars.json

```diff
@@ -35,6 +35,13 @@
         }
       ]
     },
+    {
+      "Data Flow": [
+        "concepts/data-flow/near-data-flow",
+        "concepts/data-flow/token-balance",
+        "concepts/data-flow/token-transfer-flow"
```

Comment:
from the text, it looks like token-transfer-flow goes before token-balance

Reply:
fixed